library/rental.rb

```diff
@@ -0,0 +1,27 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.rentals << self
+
+    @book = book
+    book.rentals << self
+  end
+
+  def to_s
+    "Date: #{@date}, Book: #{@book}, Person: #{@person}"
+  end
+
+  def to_json(*args)
+    {
+      JSON.create_id => self.class.name,
+      'date' => @date,
+      'book_title' => @book,
+      'person_id' => @person
+    }.to_json(*args)
+  end
+end
```

Comment:
- [ ] The previous reviewer pointed out here that the data is not preserved. Unfortunately, this issue is not addressed and all of the data (people, books, and rentals) are not preserved after being created by the user. In addition, while the app runs it should parse data from the JSON files but currently, it is using the data in the current session. Kindly fix these issues. :pray:

Reply:
Hi. I must have to disagree with you about the data preserved. All data are preserved. We uploaded the files with some data to make more "easy" for you to test, and of course, you're able to create new data. And I would like to add more information on this project request: **On exit** from your app save all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly (e.g. rentals.json)" It's mean to save the data in files after exiting the program. I would like to rest you to test again and change from "Change request" to approved.